perfkitbenchmarker/vm_util.py

```diff
@@ -233,7 +238,7 @@ def _ExecuteThreadCall(target_arg_tuple, call_id, queue, parent_log_context,
     queue.put(ThreadCallResult(call_id, None, traceback.format_exc()))
 
 
-def RunParallelThreads(target_arg_tuples, max_concurrency):
+def RunParallelThreads(target_arg_tuples, max_concurrent_threads=200):
```

Comment:
The `max_concurrent_threads` parameter of `RunThreaded` is passed directly as the second argument to `RunParallelThreads` and I thought it was strange that they had different names.  I can change the name back, or I can change the `RunThreaded` parameter to `max_concurrency` to match.